judgequeuer.py

```diff
@@ -41,9 +42,22 @@ async def main():
 
     logger.info("JudgeQueuer ready")
 
+    # await send_test_submission()
```

Comment:
Remove

Reply:
Why? Could still be nice for testing without the website

Reply:
Ok